coder-sdk/request.go

```diff
@@ -43,6 +43,9 @@ func (c Client) request(ctx context.Context, method, path string, in interface{}
 	if err != nil {
 		return nil, xerrors.Errorf("create request: %w", err)
 	}
+	if config.Query != nil {
```

Comment:
we should do this above `eq, err := http.NewRequestWithContext(`, and add `path` to url.Path`